packages/plugin-coinbase/src/plugins/trade.ts

```diff
@@ -179,22 +195,35 @@ export const executeTradeAction: Action = {
                 );
                 return;
             }
+            let amountInCurrency = amount
+            try {
+                if (side === "SELL") {
+                    const priceInUSD = await getPrice(runtime, sourceAsset);
+                    await new Promise(resolve => setTimeout(resolve, 5000));
+                    elizaLogger.info("PriceInUSD:", priceInUSD);
+                    amountInCurrency = parseFloat(((1 / priceInUSD) * amountInCurrency).toFixed(7));
```

Comment:
I haven't run it on my machine but are we sure about this equation? I'd rather say
amountInCurrency = parseFloat((priceInUSD * amountInCurrency).toFixed(7));

Reply:
We are sure as I have transfered and it worked as expected got $0.10 worth of ETH etc